packages/applications-service-api/Dockerfile

```diff
@@ -12,15 +12,16 @@ ADD api api
 ADD src src
 ADD __tests__ __tests__
 ADD jest.config.js jest.config.js
+ADD .eslintrc .eslintrc
 ADD data data
 
 ADD package.json package.json
 ADD .sequelizerc .sequelizerc
 ADD package-lock.json package-lock.json
 
-RUN npm i
-RUN npm run documentation
-RUN npm prune --production
+RUN npm i --production
+RUN npm run lint
+RUN npm run documentation; echo $? > /npm.exitcode
```

Comment:
This looks like a merge problem @lvnhmd  - I removed the `npm.exitcode` approach from the Dockerfiles

Reply:
@cpcundill - Yes, I had a conflict (which I was afraid of), I seem to have missed this, what is the change you are requesting if you have already fixed it ?